Products/CMFPlone/browser/search.py

```diff
@@ -183,10 +184,27 @@ def filter_types(self, types):
         plone_utils = getToolByName(self.context, "plone_utils")
         if not isinstance(types, list):
             types = [types]
-        return plone_utils.getUserFriendlyTypes(types)
+
+        # Get the vocabulary
+        vocab_factory = self._portal_types_vocabulary()
+        if vocab_factory is None:
```

Comment:
I think it's better to leave this test out. If someone removes the utility, the CI tests better fail and the person who removes it better analyzes the consequences of the removal.

Reply:
You can test both PRs together at:

https://jenkins.plone.org/job/pull-request-6.1-3.10/build?delay=0sec

Reply:
okay.. removed the test